Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -43,9 +43,6 @@ ControllerInterface g_controller_interface;
 //
 void ControllerInterface::Initialize(void* const hwnd)
 {
-	if (m_is_init)
-		return;
```

Comment:
Yes, I was going to do that, but this way, it allows multiple init's but will not shutdown unless it has been initialised.